cobc/cobc.h

```diff
@@ -353,11 +358,14 @@ struct list_files {
 
 extern struct list_files	*cb_current_file;
 
-extern enum cb_format		cb_source_format;
 #if 0 /* ancient OSVS registers that need special runtime handling - low priority */
 extern enum cb_current_date	current_date;
 #endif
+extern int			cb_indicator_column;
 extern int			cb_text_column;	/* end of area B (in single-byte characters) */
+extern int			cb_floating_area_b; /* whether indicator is optional */
```

Comment:
Do we need those two variables in cobc.c for listing purposes? If not move to pplex.l and drop the `cb_` prefix.